Configuration/PyReleaseValidation/python/relval_Run4.py

```diff
@@ -36,6 +36,9 @@
 numWFIB.extend([31234.0]) #Run4D114
 numWFIB.extend([32034.0]) #Run4D115
 
+# Temporary placement for LST workflow to workaround PR conflicts - to be formatted and placed in an upcoming PR
+numWFIB.extend([24834.703,24834.704]) #2026D98 LST tracking (initialStep+HighPtTripletStep only): CPU, GPU
```

Comment:
Just a note that the comment should be Run4D98, not 2026D98 anymore. Not important to fix now, but if you will do something else, please include this.

Reply:
@srimanob than you for your comments. 
Is it OK to apply this and https://github.com/cms-sw/cmssw/pull/45117#discussion_r1844143282 in a followup PR?
This is mainly to avoid a cycle of signature re-collection.

I added your comments to the todo list that we've already committed to add in followup PR(s) https://github.com/SegmentLinking/cmssw/issues/75#issuecomment-2479851825